devicetypes/smartthings/zwave-switch-generic.src/README.md

```diff
@@ -4,8 +4,10 @@
 
 Works with: 
 
-* [Leviton Appliance Module (DZPA1-1LW)](https://support.smartthings.com/hc/en-us/articles/205881176-Leviton-Appliance-Module-DZPA1-1LW-)
+* [Leviton Appliance Module (DZPA1-1LW)](http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=DZPA1-1LW)
 * [GE Plug-In Outdoor Smart Switch (GE 12720) (Z-Wave)](https://support.smartthings.com/hc/en-us/articles/200903080-GE-Plug-In-Outdoor-Smart-Switch-GE-12720-Z-Wave-)
+* [Leviton Outlet (DZR15-1LZ)](http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=DZR15-1LZ)
+* [Leviton Switch (DZS15-1LZ)](http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=DZS15-1LZ)
```

Comment:
Should we link to WWST page? 
https://www.smartthings.com/works-with-smartthings/switches-and-dimmers/leviton-switch